config/install/node.type.stanford_publication.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - menu_ui
+    - node_revision_delete
+third_party_settings:
+  menu_ui:
+    available_menus: {  }
```

Comment:
Just checking that no menu options were intentional.

Reply:
I haven't seen any requirement for menu linking at this time.

Reply:
Ty.